app/validators/employment_details/answers_validator.rb

```diff
@@ -0,0 +1,43 @@
+module EmploymentDetails
+  class AnswersValidator
```

Comment:
Is there a reason this one isn't inheriting from BaseAnswerValidator?

Reply:
That method is based on Active model validator which cannot support more complex memoization etc.